api/src/main/resources/org/openmrs/module/commonreports/sql/disbursement.sql

```diff
@@ -73,7 +73,7 @@ SELECT
             AND o_f.concept_id = (
                 SELECT concept_id 
                 FROM concept 
-                WHERE uuid LIKE '758b9dd8-b6d0-4ac2-b245-0e7bffb4693a'
+                WHERE uuid LIKE 'c158e3b2-edf9-4e0e-9e95-d36de1a51527'
```

Comment:
Shouldn't this be a config point?

Reply:
There's very many of these for the moment because of the wide requirement for the report. Is this a report we intend to re-use with other instances, or is it quite OZ Kh specific in which case I simply use UUIDs directly.

Reply:
Good point. It's very Oz Kh specific indeed.